proto/spire/server/datastore/datastore.proto

```diff
@@ -131,13 +131,11 @@ message GetNodeSelectorsResponse {
 message ListNodeSelectorsRequest {
     // When enabled, read-only connection will be used to connect to database read instances. Some staleness of data will be observed.
     bool tolerate_stale = 1;
-    Pagination pagination = 2;
```

Comment:
Should we specify `2` as a reserved field instead, so it can't be accidentally used?

Reply:
In general that is a good idea. However, in this case, it would largely be out of policy instead of practicality because:
1) DataStore is not pluggable. There is no room for implementation mismatch.
2) ListNodeSelectors has not been included in a release.

Reply:
That's fair.